client/src/components/ProjectHeader.js

```diff
@@ -5,16 +5,20 @@ import { Box, Grid, Text } from 'grommet'
 import { Badge } from 'components/Badge'
 import { Download } from 'components/Download'
 import { Link } from 'components/Link'
+import { InfoText } from 'components/InfoText'
 import { Pill } from 'components/Pill'
+import { WarningText } from 'components/WarningText'
 import { accumulateValue } from 'helpers/accumulateValue'
 import { capitalize } from 'helpers/capitalize'
 import { formatBytes } from 'helpers/formatBytes'
 import { getReadable } from 'helpers/getReadable'
-import { InfoText } from './InfoText'
-import { WarningText } from './WarningText'
 
 export const ProjectHeader = ({ project, linked = false }) => {
   const { responsive } = useResponsive()
+  const isSampleCountEqualToDownloadableCount =
```

Comment:
I think here we should just go off of the fact that `unavailable_sample_count` is non-zero. The point of this attribute is to say "these are the samples that should have computed files but dont yet"

Reply:
Completed, thank you!